### PR TITLE
Fixing accessibility bug

### PR DESCRIPTION
**Pull request by navya9singh**
Adding unique navigation landmarks. Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/2175671

### CODE REVIEW

File: packages/typescriptlang-org/src/components/layout/Sidebar.tsx [hunk 1]
```diff
@@ -122,7 +122,7 @@ export const Sidebar = (props: Props) => {
   }
 
   return (
-    <nav id="sidebar">
+    <nav aria-label = "sidebar" id="sidebar">
```
Comment (jakebailey) [inline packages/typescriptlang-org/src/components/layout/Sidebar.tsx]:
```suggestion
    <nav aria-label="sidebar" id="sidebar">
```

File: packages/typescriptlang-org/src/components/layout/TopNav.tsx [hunk 2]
```diff
@@ -98,7 +98,7 @@ export const SiteNav = (props: Props) => {
             <span className="hide-small">TypeScript</span>
           </IntlLink>
 
-          <nav role="navigation">
+          <nav aria-label = "top menu">
```
Comment (jakebailey) [inline packages/typescriptlang-org/src/components/layout/TopNav.tsx]:
```suggestion
          <nav aria-label="top menu">
```

File: packages/typescriptlang-org/src/templates/documentation.tsx [hunk 3]
```diff
@@ -159,7 +159,7 @@ const HandbookTemplate: React.FC<Props> = (props) => {
             </div>
             {showSidebar &&
               <aside className="handbook-toc">
-                <nav className={deprecationURL ? "deprecated" : ""}>
+                <nav aria-label = "table of contents">
```
Comment (jakebailey) [inline packages/typescriptlang-org/src/templates/documentation.tsx]:
```suggestion
                <nav aria-label="table of contents">
```

File: packages/typescriptlang-org/src/templates/documentation.tsx [hunk 4]
```diff
@@ -159,7 +159,7 @@ const HandbookTemplate: React.FC<Props> = (props) => {
             </div>
             {showSidebar &&
               <aside className="handbook-toc">
-                <nav className={deprecationURL ? "deprecated" : ""}>
```
Comment (jakebailey) [inline packages/typescriptlang-org/src/templates/documentation.tsx:162]: Does removing this have any effect? It seems like what this is saying is that we should hide the ToC on deprecated pages?

Comment (navya9singh) [reply packages/typescriptlang-org/src/templates/documentation.tsx:162]: I don't really see a difference as such, and I'm not sure if we should actually hide the ToC on deprecated pages, but I can add it back if you think that's needed?

**Review by jakebailey — APPROVED** [review]

**Pull request merged**